frontend/lib/sidebar-store.ts

```
@@ -0,0 +1,29 @@
+
+export function getSidebarSnapshot() {
+  return localStorage.getItem('nimbus-sidebar-collapsed') === 'true'
+}
+
+export function getSidebarServerSnapshot() {
+  return false
+}
+
+export function setSidebarCollapsed(value: boolean) {
+  localStorage.setItem('nimbus-sidebar-collapsed', String(value))
+  sidebarListeners.forEach((l) => l())
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)